docker/Dockerfile

```
@@ -25,10 +25,19 @@ FROM rust:1.74-slim-bookworm AS builder
 ARG git_commit
 ARG target
 
-RUN apt-get update && apt-get install -y \
-    pkg-config \
-    protobuf-compiler \
-    clang
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+        pkg-config \
+        protobuf-compiler \
+        libprotobuf-dev \
+        protobuf-compiler-grpc \
+        clang \
+        libssl-dev \
+        build-essential \
+        perl &&
```

I think the trailing `&&` was a typo here:
```suggestion
        perl
```